pkg/client/logging/sysinfo_windows.go

```diff
@@ -16,6 +16,14 @@ func dupToStd(_ *os.File) interface{} {
 	return errors.New("dupToStd() is not implemented on windows")
 }
 
+func dupStd() (int, int, error) {
+	return -1, -1, errors.New("dupStd() is not implemented on windows")
+}
+
+func restoreStd(int, int) error {
+	return errors.New("restoreStd() is not implemented on windows")
+}
+
```

Comment:
Rather than having `initcontext_test.go` wrap these calls in `if runtime.GOOS != "windows" { … }`, why not just make these be safe no-ops on Windows?  And the instead of having them return ints, return a cleanup function.

So on unix something like

```go
func dupStd() (func(), error) {
	stdoutFd, err := unix.Dup(1)
	if err != nil {
		return nil, err
	}
	stderrFd, err := unix.Dup(2)
	if err != nil {
		return nil, err
	}
	return func() {
		_ = unix.Dup2(stdoutFd, 1)
		_ = unix.Dup2(stderrFd, 2)
	}, nil
}
```

and on Windows

```go
func dupStd() (func(), error) {
	return func() {}, nil
}
```

Reply:
that is definitely much better